javascript-loops/main.js

```diff
@@ -0,0 +1,68 @@
+/* exported getNumbersToTen,
+            getEvenNumbersToTwenty,
+            repeatWord,
+            logEachCharacter,
+            doubleAll,
+            getKeys,
+            getValues
+ */
+function getNumbersToTen() {
+  var numbers = [];
+  var currentNumber = 1;
+  while (currentNumber < 10) {
+    numbers.push(currentNumber);
+    currentNumber++;
+  }
+  return numbers;
+}
```

Comment:
The returned array should include `10`:

---

![image](https://user-images.githubusercontent.com/16143771/152254335-3ca9f44e-d859-4ffd-aa6d-3c76ab3709df.png)

---

![image](https://user-images.githubusercontent.com/16143771/152254379-9bfc7cec-74b5-4afe-9e0f-34b6ae3daa1b.png)